tests/test_data.py

```diff
@@ -41,16 +41,8 @@ def test_pg_is_local_access(postgresql, data):
     assert data.pg_is_local_access()
 
 
-def test_pg_get_db_info(data):
-    dbinfo = data.pg_get_db_info(None)
-    assert set(dbinfo) == {
-        "timestamp",
-        "no_xact",
-        "total_size",
-        "max_length",
-        "tps",
-        "size_ev",
-    }
+def pg_get_server_information(data):
```

Comment:
```suggestion
def test_pg_get_server_information(data):
```

(Only functions named `test_<something>` are collected as tests.)